Formula/wapm.rb

```diff
@@ -0,0 +1,35 @@
+class Wapm < Formula
+  desc "📦 WebAssembly Package Manager (CLI)"
+  homepage "https://wapm.io/"
+  url "https://github.com/wasmerio/wapm-cli/archive/0.4.3.tar.gz"
+  sha256 "0a4057217a539a013549fd2bf3913bff28f0ec01d6606ebc278c2057c6e268a2"
+  head "https://github.com/wasmerio/wapm-cli.git"
+
+  depends_on "rust" => :build
+  depends_on "wasmer" => :test
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+  end
+
+  test do
+    ENV["WASMER_DIR"] = ".wasmer"
+    ENV["WASMER_CACHE_DIR"] = "#{ENV["WASMER_DIR"]}/cache"
+    Dir.mkdir ENV["WASMER_DIR"]
+    Dir.mkdir ENV["WASMER_CACHE_DIR"]
+
+    system "#{bin}/wapm", "install", "cowsay"
+
+    expected_output = <<~'EOF'
+       _____________
+      < hello wapm! >
+       -------------
+              \   ^__^
+               \  (oo)\_______
+                  (__)\       )\/\
+                     ||----w |
+                      ||     ||
+    EOF
+    assert_equal expected_output, shell_output("#{bin}/wapm run cowsay hello wapm!")
```

Comment:
Sure. And yes, though the test directory is just temporary, right? That's used to put the Wasmer package repository (and cache) in so that it doesn't mess with the user's actual packages.

Reply:
Ah, I completely missed that you moved the cache and package dirs to the temporary build directory. I'd say that this is fine as is, but an uninstall would be a good additional test.

Reply:
Sure.